corehq/apps/builds/views.py

```diff
@@ -127,7 +127,7 @@ def import_build(request):
     build_number = request.POST.get('build_number')
 
     try:
-        SemanticVersionProperty().validate(version)
+        SemanticVersionProperty(required=True).validate(version)
```

Comment:
`required=True` is the default so this change isn't necessary

Reply:
Had to add this because of [this](https://github.com/dimagi/jsonobject/blob/010391db9daabde26051511cd974f23677504f93/jsonobject/base_properties.pyx#L110).
The value that is passed during `__init__` is different from what is passed to validate and is relevant.